magefile.go

```diff
@@ -33,6 +34,8 @@ var generatedModeFile = "create_cartrdige_template_filemodes_gen.go"
 
 var completionPath = "./completion"
 
+var build_projects_path = "test/projects"
```

Comment:
Can we create an arbitrary temporary directory and pass it to the tests?

Reply:
Done